packages/atomic/create-atomic/src/plopfile.ts

```diff
@@ -184,20 +184,27 @@ export default async function (plop: NodePlopAPI) {
           return new Promise((resolve, reject) => {
             const {project} = data as PlopData;
             const process = spawn(getPackageManager(), ['install'], {
-              stdio: ['ignore', 'ignore', 'pipe'],
+              stdio: ['ignore', 'pipe', 'pipe'],
               cwd: join(currentPath, project),
             });
 
-            let error = '';
+            let out = '';
+            process.stdout.on('data', (chunk) => {
+              out += chunk.toString();
+            });
+
+            let err = '';
             process.stderr.on('data', (chunk) => {
-              error += chunk.toString();
+              err += chunk.toString();
             });
```

Comment:
this is opinionated, so feel free to not do as such.

IMO, it's best in this kind of case to have only one variable that is 'fed in'.
The idea is that as a user, you don't see a difference between stdout and stderr usually.

Moreover, on the present case, you'll log everything in stderr, so the distinction is even less valuable.

However, programs may log things in stdout that relies on human context that has been logged in stdout previously (e.g. `console.log('hello'); console.error('world');`, if you only have a lot of things like that, it'll be tough to understand what happen)

From the example, you can see that timing is more important here (IMO still).

So in essence, I would remove `err` and replace it by `out` in the stderr listener.

Reply:
> Moreover, on the present case, you'll log everything in stderr, so the distinction is even less valuable.

I kind of knew I only had stderr before... https://github.com/coveo/cli/pull/821#discussion_r890260786